From 84393ca3139178e97b0758770cdc4e35bb1ce543 Mon Sep 17 00:00:00 2001
Message-Id: <RFC-patch-01.20-84393ca3139-20221228T175458Z-avarab@gmail.com>
In-Reply-To: <RFC-cover-00.20-00000000000-20221228T175458Z-avarab@gmail.com>
References: <RFC-cover-00.20-00000000000-20221228T175458Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 25 Nov 2022 15:53:14 +0100
Subject: [RFC PATCH 01/20] t6021: mark as passing with SANITIZE=leak
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This test was recently added in 8c1bc2a71a7 (revision: add new
parameter to exclude hidden refs, 2022-11-17), but wasn't marked as
passing with "SANITIZE=leak".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t6021-rev-list-exclude-hidden.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/t6021-rev-list-exclude-hidden.sh b/t/t6021-rev-list-exclude-hidden.sh
index 32b2b094138..11c50b7c0dd 100755
--- a/t/t6021-rev-list-exclude-hidden.sh
+++ b/t/t6021-rev-list-exclude-hidden.sh
@@ -2,6 +2,7 @@
 
 test_description='git rev-list --exclude-hidden test'
 
+TEST_PASSES_SANITIZE_LEAK=true
 . ./test-lib.sh
 
 test_expect_success 'setup' '
-- 
2.39.0.1153.gb0033028ca9

